code_src/staking/dot/accounting/accountingModule.py

```diff
@@ -0,0 +1,25 @@
+import json
+from Logger import myLogger
+from substrateinterface import Keypair
+
+
+class Accounting(object):
+    def __init__(self):
+        self.name = "Accounting"
+        self.logger = myLogger(self.name)
+
+        self.logger.info("Starting accounting Program.")
+
+    def createMnemonic(self):
+        return self.logger.info(Keypair.generate_mnemonic())
+
+    def createKeypair(self, mnemonic):
+        return self.logger.info(Keypair.create_from_mnemonic(mnemonic))
+
+    def accountInfos(self, active_substrate, ss58_address):
+        return self.logger.info(active_substrate.query('System', 'Account', params=[ss58_address]).value)
+
+    def create_account(self) -> json:
```

Comment:
rename to createAccount, follow consistent same naming convention (i.e. function names are either fooBar or foo_bar, but don't mix unless there's a reason)

Reply:
now my only concern is the core functionality after that i will clean every word in the project but i will use fooBar from now.